src/network/bpf/sysctl_monitor/sysctl-monitor.bpf.c

```diff
@@ -89,6 +85,17 @@ int sysctl_monitor(struct bpf_sysctl *ctx) {
         we.pid = bpf_get_current_pid_tgid() >> 32;
         we.cgroup_id = bpf_get_current_cgroup_id();
 
+        r = bpf_current_task_under_cgroup(&cgroup_map, 0);
+
+        /* Ignore events generated by us */
+        if (r == 1)
+                return 1;
+
+        if (r < 0) {
```

Comment:
Not a big matter, but we usually check error at first, and do not insert any blank line between function call and error check. So, please do something like the following:
```c
r = bpf_current_task_under_cgroup(...);
if (r < 0) {
        we.errorcode = r;
        goto send_event;
}
if (r == 1)
        return 1; /* Ignore events generated by us */
```